### PR TITLE
Move to the official level event names

### PR DESCRIPTION
**Pull request by basaigh**

### CODE REVIEW

File: effects.json [hunk 1]
```diff
@@ -1,99 +1,94 @@
...
+  "SOUND_DISPENSER_PROJECTILE_LAUNCH": {
     "data": 1000,
     "name": "SOUND_CLICK",
     "type": "soundLevel"
   },
-  "ENTITY_ENDEREYE_LAUNCH": {
-    "identifier": "minecraft:player",
-    "name": "THROW",
-    "type": "soundEvent"
-  },
```
Comment (onebeastchris) [inline effects.json:29]: why was this removed?

**Review by onebeastchris — APPROVED** [review]

File: effects.json (hunk 1 above)
Comment (basaigh) [reply effects.json:29]: It's no longer present in the game. I can re-add it, however it would never be used.

Comment (onebeastchris) [reply effects.json:29]: Ah, I see. Makes sense to remove it; thanks for the clarification!
